lib/session.js

```diff
@@ -214,6 +228,14 @@ module.exports = function (options) {
    *   if session is modified, update cookie and store
    */
   function *refreshSession (session, originalHash, isNew) {
+    if(this._sessionSave === false) {
+      return debug('session save disabled');
+    }
+    if(this._sessionSave === true) {
+      debug('session save forced');
+      return yield saveNow.call(this, this.sessionId, session);
```

Comment:
Hmm...should probably preserve the session destroying behavior.  I'll play with pushing it below the delete block.